app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.tsx

```diff
@@ -233,21 +240,25 @@ function LloydGeorgeUploadPage() {
         setDocuments([]);
         setStage(LG_UPLOAD_STAGE.SELECT);
     };
+
     const startIntervalTimer = (uploadDocuments: Array<UploadDocument>) => {
-        return window.setInterval(() => {
-            uploadDocuments.forEach(async (document) => {
-                try {
-                    await updateDocumentState({
-                        document,
-                        uploadingState: true,
-                        baseUrl,
-                        baseHeaders,
-                    });
-                } catch (e) {}
-            });
+        return window.setInterval(async () => {
+            await setDocumentUploadingState(uploadDocuments, true);
```

Comment:
```suggestion
        return window.setInterval(() => {
            void setDocumentUploadingState(uploadDocuments, true);
```
Maybe better not to use `async` function as the input of setInterval, as there's nothing that will wait for the returned promise.